mitosheet/mitosheet/ai/recon.py

```diff
@@ -135,6 +135,16 @@ def get_modified_dataframe_recon_data(old_df: pd.DataFrame, new_df: pd.DataFrame
     old_columns = old_df.columns.to_list()
     new_columns = new_df.columns.to_list()
 
+    # If there are duplicated columns in the new dataframe, we throw an error
+    # as we don't support this elsewhere in the codebase
+    if len(new_columns) != len(set(new_columns)):
```

Comment:
I got this error, which is not the error generated here, but also not an error I've gotten before when using Mito AI. I've never seen it generate a modal before. Do you think that is caused by this change?

Some things to fix:
1. Instead of the modal, we should handle the error in place like we usually do with Mito AI
2. ^ should also make it so that the prompt and result are displayed. See in this example the prompt disappeared and we are back to the example. 

<img width="1074" alt="Screenshot 2023-05-30 at 1 06 51 PM" src="https://github.com/mito-ds/monorepo/assets/18709905/0b342f17-1951-42e6-81f9-aa0f98daa7d5">